src/styles/utils/variables.scss

```diff
@@ -0,0 +1,6 @@
+$card-width: 198px;
```

Comment:
The card width does not match the requirement. It should be set to 200px according to the task description.